layouts/partials/head.html

```diff
@@ -17,7 +17,11 @@
   <link rel="icon" href="{{ `images/favicon.png` | absURL }}" type="image/x-icon">
 
   {{ "<!-- fonts -->" | safeHTML }}
-  <link href="https://fonts.googleapis.com/css?family={{replace .Site.Params.font_family ' ' '+' | title}}:300,400,700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
Is this a performance gain for loading font caches or something earlier? Sorry if it is an obvious question, I am curious. 🙊

Reply:
@jwflory really has nothing much to do with performance. So when "Open+Sans" is used it is not being parsed correctly therefore an error shows in the console.
![inventory error](https://user-images.githubusercontent.com/56026636/116773984-23862980-aa51-11eb-9fa7-6885713c4311.png)

I just removed the changes I made in this file and the config file. We could figure out a better way to tackle this :)

Reply:
@Idadelveloper Good catch here. I didn't notice this bug until now. You could open an issue on [unicef/inventory](https://github.com/unicef/inventory) so we could talk through how best to fix that one.